homeworks/hw1-dom-node/Bad-Company/lib/attr_class.rb

```
@@ -0,0 +1,25 @@
+class AttrClass < Attribute
+
+  def initialize(value)
+    super(:class, process_value(value))
+  end
+
+  def append(value)
+    @value << process_value(value)
+  end
+
+  def to_s
+    "#{@name}=\"#{@value.join(' ')}\""
+  end
+
+  private
+
+  def process_value(value)
+    case value
+      when String
+        value.split(' ')
+      else
+        value
+    end
```

What about `Hash` or any other type of a value?
You must handle only `String` and `Array`. Otherwise you should raise an exception.